webcompat/templates/home-page/how-it-works.html

```diff
@@ -0,0 +1,40 @@
+<section class="grid">
+  <article class="grid-row">
+    <div class="grid-cell x2">
+      <h2 class="headline-1 highlight">What is this project about?</h2>
+      <p>
+        People expect websites to work regardless of the device or browser they are using. If they are behaving differently, that is most likey a web compatibility bug we are looking for.
+      </p>
+    </div>
+<div class="grid-cell x1">
+        <h3 class="headline-4">Join Slack!</h3>
+        <p>Come and join our web compat community.</p>
+        <a href="./contact" class="button button-primary">Let's chat</a>
+      </div>
+  </article>
+</section>
+
+<section class="grid">
+  <article class="grid-row">
+    <div class="grid-cell x1">
+      <h3 class="headline-4">Meet the team</h2>
+      <p>
+        Who are the people behind the web compat project? <br /><br/>
+
+        Find out and <a href="./about" title="meet the wompats">meet the Wompats</a>.
```

Comment:
I know you've used this term before, and I don't have any strong objections here -- just a note: npm has a wombat as their mascot. So we should be aware of that at least, in case someone thinks we're ripping them off or something. I guess only a real issue if we start making wombat stickers or logos? 

https://medium.com/npm-inc/tagged/wombat

Reply:
Yes, I'm aware of that. The people from npm I talked to about it, didn't see any issues and liked it.

Reply:
OK. I would prefer "meet the team", but it can be changed later.